apps/api/pom.xml

```diff
@@ -12,24 +12,58 @@
 	<artifactId>api</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
 	<name>api</name>
-	<description>An API for treecreate web store. Utilizes Springboot and JPA (MySQL)</description>
+	<description>An API for Treecreate web store. Utilizes Springboot and JPA (MySQL)</description>
 
 	<properties>
 		<java.version>11</java.version>
+        <groovy.version>3.0.2</groovy.version> <!-- required for rest-assured or it defaults to an oder one -->
 	</properties>
 
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
 
+        <!-- Test Scope -->
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
```

Comment:
JUnit is included in the spring-boot-starter-test dependency, no need to include it explicitly